pkg/executor/push.go

```diff
@@ -109,6 +109,13 @@ func DoPush(image v1.Image, opts *config.KanikoOptions) error {
 	t := timing.Start("Total Push Time")
 	var digestByteArray []byte
 	var builder strings.Builder
+
+	dir := strings.Split(opts.DigestFile, "/")
+	parentDir := strings.Join(dir[:len(dir)-1], "/")
```

Comment:
Lines 113 and 114 can be replaced with `parentDir := filepath.Dir(opts.DigestFile)`

Reply:
updated